docs/src/writing-tests-java.md

```diff
@@ -0,0 +1,94 @@
+---
+id: writing-tests
+title: "Writing Tests"
+---
+
+Playwright assertions are created specifically for the dynamic web. Checks are automatically retried until the necessary conditions are met. Playwright comes with [auto-wait](./actionability.md) built in meaning it waits for elements to be actionable prior to performing actions. Playwright provides an [expect](./test-assertions.md) function to write assertions.
```

Comment:
We need an intro before the assertions and the assertions should probably go into its own section.

Reply:
Lets keep it for now? Its the same like in other langs